src/databrowser.js

```diff
@@ -132,13 +131,14 @@ define([
                 state: {opened: !node.recursive && children.length <= MAX_OPEN_NODE},
                 children: children,
                 data: {
-                    handleDrop: _.partial(handleDrop, path, node.sourceInfo),
+                    handleDrop: _.partial(handleDrop, node, node.sourceInfo),
                     getNodes: node.recursive ? getNodes : null,
                 }
             };
         }
-        function handleDrop(path, info, target) {
-            var widget = widgets.util.getWidget(target, vellum);
+        function handleDrop(node, info, target) {
+            var widget = widgets.util.getWidget(target, vellum),
+                path = widget.mug.form.richText && node.hashtag || node.xpath;
```

Comment:
I'm guessing I wrote that without even thinking (`x and y or z` used to be the idiomatic way to do an inline conditional in Python, not without its gotchas (`true && "" || "wrong" === "wrong"`), so not recommended). Fixed in 1daecca